frontend/src/metabase/query_builder/reducers.js

```diff
@@ -548,17 +554,23 @@ export const currentState = handleActions(
   null,
 );
 
-export const dashboardId = handleActions(
+export const dashboardControls = handleActions(
```

Comment:
nit: not sure if `dashboardControls` is the best name here. "Controls" makes me think about interactable UI elements or something dynamic. Perhaps something like `parentDashboard` or just `dashboard` would work better?

Reply:
Agree, fixed